md-dao/src/main/java/com/wipro/ats/bdre/md/dao/JobDAO.java

```diff
@@ -283,21 +285,17 @@
                             BatchConsumpQueue batchConsumpQueueUpdated = (BatchConsumpQueue) updateTargetBatchCriteria.list().get(0);
                             batchConsumpQueueUpdated.setBatchByTargetBatchId(batch);
                             session.update(batchConsumpQueueUpdated);
-                            //BCQObjectList.add(batchConsumpQueue);
                         }
-                        //queueIdMap.put(blankBCPProcess.getProcessId(),BCQObjectList);
-
                     }
                 }
 
-                Criteria nullBCPCriteria = session.createCriteria(Process.class).add(Restrictions.eq("process", parentProcess))
-                        .add(Restrictions.eq("deleteFlag", false)).add(Restrictions.ne("enqueuingProcessId", 0))
-                        .add(Restrictions.isNull("batchCutPattern"));
+                Criteria nullBCPCriteria = session.createCriteria(Process.class).add(Restrictions.eq(PROCESS, parentProcess))
+                        .add(Restrictions.eq(DELETEFLAG, false)).add(Restrictions.ne(ENQUEUINGPROCESSID, 0))
+                        .add(Restrictions.isNull(BATCHCUTPATTERN));
                 for (Object nullBDPObject : nullBCPCriteria.list()) {
                     Process blankBCPProcess = (Process) nullBDPObject;
-                    Criteria entriesForNullBCPCriteria = session.createCriteria(BatchConsumpQueue.class).add(Restrictions.eq("process", blankBCPProcess));
-                    // List<BatchConsumpQueue> BCQObjectList = new ArrayList<BatchConsumpQueue>();
-                    if (entriesForNullBCPCriteria.list().size() > 0) {
+                    Criteria entriesForNullBCPCriteria = session.createCriteria(BatchConsumpQueue.class).add(Restrictions.eq(PROCESS, blankBCPProcess));
+                    if (!entriesForNullBCPCriteria.list().isEmpty()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)